test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,42 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+public class UtilsTest {
+	
+	@Test
+	public void nullObject_returnsTrue() {
+		Integer a = null;
+		assertTrue(Utils.isAnyNull(a));
+	}
+	
+	@Test
+	public void notNullObject_returnsFalse(){
```

Comment:
the test name should have 3 parts `<method being tested>_<what is being tested>_<what is the expected return value>`
you can eliminate part 3 and 2 only if you have broadened the scope of the test to include testing multiple things.